pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/kafka/KafkaHighLevelStreamProviderConfig.java

```diff
@@ -41,6 +41,10 @@
     defaultProps.put("group.id", groupId);*/
     defaultProps.put("zookeeper.session.timeout.ms", "30000");
     defaultProps.put("zookeeper.sync.time.ms", "200");
```

Comment:
dont know why this is 200, we should set this higher like 30 sec,
Also add zookeeper.connection.timeout.ms	and set it to 10 seconds